test/hotspot/jtreg/gtest/NativeHeapTrimmerGtest.java

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 2023, 2024 Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2023, 2024 Red Hat, Inc. All rights reserved.
+ * Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, 2024, Red Hat, Inc. All rights reserved.
```

Comment:
Please note the comma remark only applied to Oracle copyright notices. I have no idea what the rules are for Red hat notices.

Reply:
We seem to be doing this too.